VirtoCommerce.Domain/Catalog/Model/CatalogProduct.cs

```diff
@@ -111,5 +112,64 @@ public class CatalogProduct : AuditableEntity, ILinkSupport, ISeoSupport, IHasOu
         #region IHasOutlines members
         public ICollection<Outline> Outlines { get; set; }
         #endregion
+
+        public virtual void ReduceDetails(string responseGroup)
+        {
+            var productResponseGroup = EnumUtility.SafeParseFlags(responseGroup, ItemResponseGroup.ItemLarge);
+
+            if (!productResponseGroup.HasFlag(ItemResponseGroup.ItemAssets))
+            {
+                Assets = null;
+            }
+
+            if (!productResponseGroup.HasFlag(ItemResponseGroup.ItemProperties))
+            {
+                Properties = null;
```

Comment:
Maybe we could clear PropertyValues (need to verify if they are cleared in platform-core).

Reply:
it's ok with PropertyValues.
Added missed field